sqlite-android/src/main/java/io/requery/android/database/sqlite/RequerySQLiteOpenHelperFactory.java

```diff
@@ -67,4 +91,8 @@ public void onCorruption(SQLiteDatabase db) {
             callback.onCorruption(db);
         }
     }
-}
+
+    public interface RequeryConfigurationOption {
```

Comment:
This inner class can just be named `ConfigurationOption` I don't think the requery prefix adds much since it's already in a class prefixed with that.